app/controllers/application_controller.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationController < ActionController::Base
   def current_user
-    @user = User.first
+    User.first
```

Comment:
- [ ] [OPTIONAL] I like the description you wrote for this PR. It's really well written. Personal suggestion, have you used `rails db:seed` before? If not, here is the chance to test it. You can look through about it in [here](https://blog.devgenius.io/how-to-seed-with-ruby-on-rails-%EF%B8%8F-1d2dceda3e7d#:~:text=%23%20db/seed.rb,puts%20%22Seeding%20done.%22).

Reply:
@Rustamxon7 I have not seen this before thanks for sharing! I appreciate the extra knowledge :)